.github/workflows/future-release-branch-labeler.yml

```diff
@@ -0,0 +1,34 @@
+name: Future Release Branch Pull Request Labeler
+
+on:
+  pull_request_target:
+    branches:
+      - dev-[0-9].[0-9]+
+    types:
+      - opened
+      - reopened
+
+jobs:
+  add-milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+          # determine the target release from the branch that this PR targets
+          # ignore first 4 characters ("dev-")
+          milestone=${GITHUB_BASE_REF:4}
+
+          # grab the number associated with milestone not the name of milestone
+          curl --request GET \
+          --url 'https://api.github.com/repos/${{ github.repository }}/milestones' \
+          --header 'Authorization: token ${{ secrets.FUTURE_RELEASE_BRANCH_LABEL_TOKEN }}' \
+          --header 'Accept: application/vnd.github.v3+json' | \
+          jq --arg milestone "$milestone" '.[] | select(.title==$milestone).number' > milestone_number.txt
+
+          milestone_number="$(cat milestone_number.txt)"
+
+          curl --request PATCH \
+          --url ${{ github.event.pull_request.issue_url }} \
+          --header 'Authorization: token ${{ secrets.FUTURE_RELEASE_BRANCH_LABEL_TOKEN }}' \
+          --header 'Accept: application/vnd.github.v3+json' \
```

Comment:
I am looking at this API guide, https://docs.github.com/en/free-pro-team@latest/rest/reference/issues#milestones.
Is the `data` parameter the same as the `title` parameter?
```
-d '{"title":"$milestone_number"}'
```

Reply:
This request is being made to `issue` endpoint, not the `milestone` endpoint, which has different types of parameters. (https://docs.github.com/en/free-pro-team@latest/rest/reference/issues)

Reply:
OK